_participants/kjeldschmidt.md

```diff
@@ -0,0 +1,7 @@
+---
+---
+name: Kjeld
+pronouns: he/him
+image: https://drive.google.com/uc?export=view&id=1QqwXoxpqhk6bcnK7gIQMWkk3a3lVUa85
+linkedin: https://www.linkedin.com/in/andreasbraeu/
```

Comment:
Please provide the correct link

Reply:
Yes, very sorry for this and thanks for the correction!